### DISCUSSION TITLE
To run an EC2 instance with EJBCA and a MariaDB RDS database instance on AWS,

### DISCUSSION THREAD

**Discussion by Babacar98** [Keyfactor/ejbca-ce#706] [Q&A]
I used an EC2 instance in which I connected another RDS MariaDB database instance, then I created a containers directory and a docker-compose.yml file. But when I run the command "$ docker compose logs -f", I see this output, but I don't see the URL                                              

  ejbca          | 2024-11-11 11:43:09,578+0000 INFO  [org.cesecore.audit.impl.log4j.Log4jDevice] (ServerService Thread Pool -- 75) 2024-11-11 11:43:09+00:00;EJBCA_STARTING;SUCCESS;SERVICE;EJBCA;Application internal;;ejbca-node1;;msg=Init, EJBCA 8.3.2 Community (d570abdfbd2eac6760c3050f9e64308366fc4a51) startup.
ejbca          | 2024-11-11 11:43:09,588+0000 INFO  [org.cesecore.audit.impl.log4j.Log4jDevice] (ServerService Thread Pool -- 75) 2024-11-11 11:43:09+00:00;LOG_MANAGEMENT_CHANGE;VOID;SECURITY_AUDIT;CORE;Application internal;;;;msg=No integrity protected security audit logger devices configured.
ejbca          | 2024-11-11 11:43:09,604+0000 INFO  [org.reflections.Reflections] (ServerService Thread Pool -- 77) Reflections took 113 ms to scan 1 urls, producing 9 keys and 22 values
ejbca          | 2024-11-11 11:43:09,617+0000 INFO  [org.cesecore.dbprotection.ProtectedData] (ServerService Thread Pool -- 75) No database integrity protection available in this version of EJBCA.
ejbca          | 2024-11-11 11:43:09,691+0000 INFO  [org.jboss.resteasy.resteasy_jaxrs.i18n] (ServerService Thread Pool -- 77) RESTEASY002205: Adding provider class org.ejbca.ui.web.rest.api.config.ObjectMapperContextResolver from Application class org.ejbca.ui.web.rest.api.RestApiApplication
ejbca          | 2024-11-11 11:43:09,691+0000 INFO  [org.jboss.resteasy.resteasy_jaxrs.i18n] (ServerService Thread Pool -- 77) RESTEASY002205: Adding provider class org.ejbca.ui.web.rest.api.exception.ValidationExceptionMapper from Application class org.ejbca.ui.web.rest.api.RestApiApplication
ejbca          | 2024-11-11 11:43:09,694+0000 INFO  [org.jboss.resteasy.resteasy_jaxrs.i18n] (ServerService Thread Pool -- 77) RESTEASY002200: Adding class resource org.ejbca.ui.web.rest.api.resource.swagger.SystemRestResourceSwagger from Application class org.ejbca.ui.web.rest.api.RestApiApplication
ejbca          | 2024-11-11 11:43:09,694+0000 INFO  [org.jboss.resteasy.resteasy_jaxrs.i18n] (ServerService Thread Pool -- 77) RESTEASY002200: Adding class resource org.ejbca.ui.web.rest.api.resource.swagger.CertificateRestResourceV2Swagger from Application class org.ejbca.ui.web.rest.api.RestApiApplication
ejbca          | 2024-11-11 11:43:09,695+0000 INFO  [org.jboss.resteasy.resteasy_jaxrs.i18n] (ServerService Thread Pool -- 77) RESTEASY002205: Adding provider class org.ejbca.ui.web.rest.api.exception.IllegalWildCardSyntaxExceptionWrapper from Application class org.ejbca.ui.web.rest.api.RestApiApplication
ejbca          | 2024-11-11 11:43:09,698+0000 INFO  [org.jboss.resteasy.resteasy_jaxrs.i18n] (ServerService Thread Pool -- 77) RESTEASY002200: Adding class resource org.ejbca.ui.web.rest.api.resource.swagger.CertificateRestResourceSwagger from Application class org.ejbca.ui.web.rest.api.RestApiApplication
ejbca          | 2024-11-11 11:43:09,698+0000 INFO  [org.jboss.resteasy.resteasy_jaxrs.i18n] (ServerService Thread Pool -- 77) RESTEASY002205: Adding provider class org.ejbca.ui.web.rest.api.config.ExceptionHandler from Application class org.ejbca.ui.web.rest.api.RestApiApplication
ejbca          | 2024-11-11 11:43:09,699+0000 INFO  [org.jboss.resteasy.resteasy_jaxrs.i18n] (ServerService Thread Pool -- 77) RESTEASY002200: Adding class resource org.ejbca.ui.web.rest.api.resource.swagger.CaRestResourceSwagger from Application class org.ejbca.ui.web.rest.api.RestApiApplication
ejbca          | 2024-11-11 11:43:09,744+0000 INFO  [org.ejbca.core.ejb.authorization.AuthorizationSystemSessionBean] (ServerService Thread Pool -- 75) Roles or CAs exist, not initializing Super Administrator Role
ejbca          | 2024-11-11 11:43:09,994+0000 INFO  [org.cesecore.certificates.certificate.CertificateStoreSessionBean] (ServerService Thread Pool -- 75) Setting isUniqueCertificateSerialNumberIndex to: true
ejbca          | 2024-11-11 11:43:10,164+0000 INFO  [org.cesecore.audit.impl.log4j.Log4jDevice] (ServerService Thread Pool -- 75) 2024-11-11 11:43:10+00:00;ACCESS_CONTROL;SUCCESS;ACCESSCONTROL;CORE;Internal upgrade;;;;resource0=/system_functionality/edit_systemconfiguration
ejbca          | 2024-11-11 11:43:10,183+0000 INFO  [org.cesecore.audit.impl.log4j.Log4jDevice] (ServerService Thread Pool -- 75) 2024-11-11 11:43:10+00:00;SYSTEMCONF_EDIT;SUCCESS;GLOBALCONF;CORE;Internal upgrade;;;;msg=Saved global configuration with id UPGRADE.
ejbca          | 2024-11-11 11:43:10,218+0000 INFO  [org.ejbca.core.ejb.upgrade.UpgradeSessionBean] (ServerService Thread Pool -- 75) Database content version: 8.3.2, current application version: 8.3.2 -> Upgrade is not needed.
ejbca          | 2024-11-11 11:43:10,256+0000 INFO  [org.cesecore.certificates.ocsp.cache.OcspSigningCache] (ServerService Thread Pool -- 75) No default responder was defined. OCSP requests for certificates issued by unknown CAs will return "unauthorized" as per RFC6960, Section 2.3
ejbca          | 2024-11-11 11:43:10,268+0000 INFO  [org.cesecore.certificates.certificate.CertificateStoreSessionBean] (ServerService Thread Pool -- 75) Reloading CA certificate cache.
ejbca          | 2024-11-11 11:43:10,325+0000 INFO  [org.cesecore.certificates.certificate.CertificateStoreSessionBean] (ServerService Thread Pool -- 75) Reloaded CA certificate cache with 0 certificates
ejbca          | 2024-11-11 11:43:10,458+0000 INFO  [org.ejbca.core.ejb.ocsp.OcspResponseCleanupSessionBean] (ServerService Thread Pool -- 75) OCSP clean up job is disabled.
ejbca          | 2024-11-11 11:43:10,640+0000 INFO  [org.owasp.csrfguard.config.properties.PropertyUtils] (ServerService Thread Pool -- 88) The 'org.owasp.csrfguard.PRNG.Provider' property was not defined, using 'SUN' as default value. %n
ejbca          | 2024-11-11 11:43:10,655+0000 INFO  [org.owasp.csrfguard.config.properties.PropertyUtils] (ServerService Thread Pool -- 88) The 'org.owasp.csrfguard.TokenHolder' property was not defined, using 'org.owasp.csrfguard.token.storage.impl.InMemoryTokenHolder' as default value. %n
ejbca          | 2024-11-11 11:43:12,411+0000 WARN  [io.undertow.servlet] (ServerService Thread Pool -- 83) UT015020: Path /* is secured for some HTTP methods, however it is not secured for [HEAD, POST, GET]
ejbca          | 2024-11-11 11:43:15,685+0000 INFO  [org.primefaces.webapp.PostConstructApplicationEventListener] (ServerService Thread Pool -- 88) Running on PrimeFaces 12.0.0
ejbca          | 2024-11-11 11:43:15,704+0000 WARN  [io.undertow.servlet] (ServerService Thread Pool -- 88) UT015020: Path /* is secured for some HTTP methods, however it is not secured for [HEAD, POST, GET]
ejbca          | 2024-11-11 11:43:16,656+0000 INFO  [org.jboss.as.server] (ServerService Thread Pool -- 37) WFLYSRV0010: Deployed "ejbca.ear" (runtime-name : "ejbca.ear")
ejbca          | 2024-11-11 11:43:16,657+0000 INFO  [org.jboss.as.server] (ServerService Thread Pool -- 37) WFLYSRV0010: Deployed "jdbc-driver.jar" (runtime-name : "jdbc-driver.jar")
ejbca          | 2024-11-11 11:43:17,301+0000 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0212: Resuming server
ejbca          | 2024-11-11 11:43:17,516+0000 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: WildFly Full 26.1.3.Final (WildFly Core 18.1.2.Final) started in 51992ms - Started 5194 of 5319 services (278 services are lazy, passive or on-demand) - Server configuration file in use: standalone.xml
ejbca          | 2024-11-11 11:43:17,575+0000 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0060: Http management interface listening on http://127.0.0.1:9990/management
ejbca          | 2024-11-11 11:43:17,576+0000 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0054: Admin console is not enabled
ejbca          | 2024-11-11 11:43:18,603+0000 INFO  [/opt/keyfactor/bin/start.sh] (process:1) Application /opt/keyfactor/appserver/standalone/deployments/ejbca.ear.deployed successfully started.
ejbca          | 2024-11-11 11:43:18,634+0000 INFO  [/opt/keyfactor/bin/start.sh] (process:1) Setting up in-bound connectivity...
ejbca exited with code 255
ubuntu@ip-*-*-*-*-~/containers$

**Comment by primetomas** [comment]
It look like it exited unexpectedly? What does "exited with code 255" mean?
